wikitemplate.md

```diff
@@ -201,15 +186,37 @@
 - [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
 - [ ] Test that hovering the cursor over a link changes the cursor into a pointer (hand)
 - [ ] Visit https://dlptest.com/ftp-test/ and ensure that basic authentication is working by clicking on ftp://ftp.dlptest.com/ and ensuring you're prompted for credentials
+- [ ] Test that WebSockets are working by ensuring http://slither.io/ runs once "Play" has been clicked.
 
-## Flash tests
-
-- [ ] Test that flash placeholder appears on http://www.homestarrunner.com
-- [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com
-
-## Autofill tests
+## Ledger
 
-- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
+- [ ] Verify wallet is auto created after enabling payments
+- [ ] Verify monthly budget and account balance shows correct BAT and USD value
+- [ ] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
+- [ ] Verify that Brave BAT wallet address can be copied
+- [ ] Verify adding funds via any of the currencies flows into BAT Wallet after specified amount of time
+- [ ] Verify adding funds to an existing wallet with amount, adjusts the BAT value appropriately
+- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
+- [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table
+- [ ] Check that disabling payments and enabling them again does not lose state
+- [ ] Upgrade from older version
+  - [ ] Verify the wallet isn't corrupted upon upgrade (balance is retained and wallet backup code isn't corrupted)
+  - [ ] Verify publishers list is not lost after upgrade
+### Ledger Media (To be verified on YouTube and Twitch) 
+  - [ ] Visit any YouTube/Twitch video in a normal/session tab and ensure the video publisher name is listed in ledger table
+  - [ ] Visit any YouTube/Twitch video in a private tab and ensure the video publisher name is not listed in ledger table
+  - [ ] Visit any live YouTube/Twitch video and ensure the time spent is shown under ledger table
+  - [ ] Visit any embeded YouTube/Twitch video and ensure the video publisher name is listed in ledger table
+  - [ ] Ensure total time spent is correctly calculated for each publisher video
+  - [ ] Ensure total time spent is correctly calculated when switching to YouTube video from an embeded video
```

Comment:
Should be ` when switching to YouTube/Twitch video` as we're testing embedded Twitch in the following case:

* `Visit any embeded YouTube/Twitch video and ensure the video publisher name is listed in ledger table`